schema/mapem/mapem_schema_2-0-0.json

```diff
@@ -243,7 +243,6 @@
                   "minItems": 1,
                   "maxItems": 16,
                   "items": {
-                    "$comment": "keep camelCase for ease of ASN.1 <> JSON translation, or use snake_case?",
```

Comment:
You should keep that comment.

Reply:
It was an oversight, I rebased so that it would not appear anymore in the original MAPEM commit.